includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1653,29 +1663,36 @@ public static function filter_tag_row_actions( $actions, WP_Term $tag ) {
 		// Only add the 'Remove' and 'Keep' links to the index page, not the individual URL page.
 		if ( 'edit-tags.php' === $pagenow ) {
 			$sanitization = self::get_validation_error_sanitization( json_decode( $term->description, true ) );
+			$is_accepted  = $sanitization['status'] & self::ACCEPTED_VALIDATION_ERROR_BIT_MASK;
+
+			$actions[ self::VALIDATION_ERROR_ACKNOWLEDGE_ACTION ] = sprintf(
+				'<a href="%s">%s</a>',
+				wp_nonce_url(
+					add_query_arg( array_merge( [ 'action' => self::VALIDATION_ERROR_ACKNOWLEDGE_ACTION ], compact( 'term_id' ) ) ),
+					self::VALIDATION_ERROR_ACKNOWLEDGE_ACTION
+				),
+				esc_html(
+					! ( $sanitization['term_status'] & self::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK ) ? __( 'Approve', 'amp' ) : __( 'Disapprove', 'amp' )
+				)
+			);
 
-			if ( self::VALIDATION_ERROR_ACK_ACCEPTED_STATUS !== $sanitization['status'] ) {
+			if ( ! $is_accepted ) {
 				$actions[ self::VALIDATION_ERROR_ACCEPT_ACTION ] = sprintf(
 					'<a href="%s">%s</a>',
 					wp_nonce_url(
 						add_query_arg( array_merge( [ 'action' => self::VALIDATION_ERROR_ACCEPT_ACTION ], compact( 'term_id' ) ) ),
 						self::VALIDATION_ERROR_ACCEPT_ACTION
 					),
-					esc_html(
-						self::VALIDATION_ERROR_NEW_ACCEPTED_STATUS === $sanitization['term_status'] ? __( 'Confirm removed', 'amp' ) : __( 'Remove', 'amp' )
-					)
+					esc_html__( 'Remove', 'amp' )
 				);
-			}
-			if ( self::VALIDATION_ERROR_ACK_REJECTED_STATUS !== $sanitization['status'] ) {
+			} elseif ( $is_accepted ) {
```

Comment:
This condition is already implied by not matching the above `! $is_accepted`.
```suggestion
			} else {
```